build/automation/var/profile/task.mk

```diff
@@ -27,6 +27,7 @@ TF_VAR_nhs_uk_api_key_key := NHS_UK_API_KEY
 
 # Lambda Security Group
 TF_VAR_lambda_security_group_name := $(PROJECT_ID)-$(ENVIRONMENT)-lambda-sg
+TF_VAR_dos_db_name := uec-dos-int-di-161
```

Comment:
uec-dos-int-di-161 is a temporary value for the mock db but it allows the lambdas to connect to the security group of the database. So this value will be changed so that we could hook up into dos or service finder db

Reply:
ah yep forgot about that